src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -41,7 +41,7 @@
                         
```

Comment:
This tool operates very quickly, processing each genome in just a few seconds. The default configuration for task scheduling is sufficient.